rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -675,11 +878,30 @@ def main():
         '--workers-wait-time',
         type=int,
         default=15,
-        help="Waiting time in minutes for the workers to be Ready after cluster installation. Default: 15 minutes")
+        help="Waiting time in minutes for the workers to be Ready after cluster installation.. If 0, do not wait. Default: 15 minutes")
```

Comment:
Somewhat unrelated but can we increase the default worker wait time? 15 is just a little to close to the average install time for me :-)

Reply:
Increased to 1 hour